src/components/home_page/MovieCard.js

```diff
@@ -4,16 +4,18 @@ import ProgressBar from './ProgressBar';
 import DescriptionMovie from './DescriptionMovie';
 
 export default function MovieCard({movieData, background}) {
-    const {id, title, releaseDate, average, posterPath, score} = movieData;
+    const {id, title, releaseDate, average, posterPath, score, mediaType} = movieData;
     return (
         <div className='movie--card'>
             <li 
-                key={id} 
+                key={id}
```

Comment:
you use the `key` on an element that is being repeated on a loop, as a map. Here is not the case.